src/client/response.rs

```diff
@@ -95,12 +95,12 @@ impl ClientResponse {
 
 impl fmt::Debug for ClientResponse {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        let res = writeln!(f, "\nClientResponse {:?} {}", self.version(), self.status());
-        let _ = writeln!(f, "  headers:");
+        writeln!(f, "\nClientResponse {:?} {}", self.version(), self.status())?
```

Comment:
Please add `;`

Reply:
Sorry. It looks like I pushed prematurely.